pkg/istio-agent/testdata/grpc-bootstrap.json

```diff
@@ -14,7 +14,33 @@
   ],
   "node": {
     "id": "sidecar~127.0.0.1~pod1.fake-namespace~fake-namespace.svc.cluster.local",
-    "metadata": {},
+    "metadata": {
+      "INSTANCE_IPS": "127.0.0.1",
+      "PILOT_SAN": [
+        "istiod.istio-system.svc"
+      ],
+      "PROV_CERT": "/Users/landow/go/src/istio.io/istio/tests/testdata/certs/pilot/root-cert.pem",
```

Comment:
seems possibly not determinstic?

Reply:
fixed - just ignoring the fields with these